go.mod

```diff
@@ -1,12 +1,70 @@
 module terraform-provider-maas
 
-go 1.16
+go 1.19
 
 require (
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.1
-	github.com/ionutbalutoiu/gomaasclient v0.0.0-20210707081625-0c69fffd5a8c
```

Comment:
Glad that we have updated to the latest gomaasclient and just something to keep in the back of our minds if there looks to be any issues.